todo_app/data/session_items.py

```diff
@@ -1,78 +1,67 @@
+import requests
```

Comment:
I expected to see `requests` in your pyproject.toml file.

Did you run `poetry add requests`? That should install the `requests` library and update pyproject.toml automatically

If you installed all your requirements with pip and then ran `flask run`, that's the wrong approach. We want a record of the project's dependencies and an easy way of installing them (`poetry install` followed by `poetry run flask run` as documented in the README)